lib/cdo/aws/cloudfront.rb

```diff
@@ -250,6 +240,64 @@ def self.config(app, reference = nil)
       end
     end
 
+    def self.get_app_config(app, behavior_method)
+      config = app == :hourofcode ? HTTP_CACHE[:pegasus] : HTTP_CACHE[app]
+      cloudfront = CONFIG[app]
+      behaviors = config[:behaviors].map do |behavior|
+        paths = behavior[:path]
+        paths = [paths] unless paths.is_a? Array
+        validate_paths paths
+        paths.map do |path|
+          behavior_method.call(behavior, path)
+        end
+      end.flatten
+      return behaviors, cloudfront, config
+    end
+
+    # Same as #config, except returns a CloudFormation JSON.
+    # `app` is a symbol containing the app name (:pegasus, :dashboard or :hourofcode)
+    def self.config_cloudformation(app, origin, aliases, ssl_cert=nil)
+      behaviors, cloudfront, config = get_app_config(app, method(:cache_behavior_cloudformation))
+      {
+        Aliases: aliases,
+        CacheBehaviors: behaviors,
+        Comment: '',
+        CustomErrorResponses: [400, 403, 404, 405, 414, 500, 501, 502, 503, 504].map do |error|
```

Comment:
Does this list have to be so specific; e.g. would you also want this behavior for a hypothetical 406 error?